### ISSUE TITLE
When updating the program overwrites publisher.zip

### ISSUE THREAD

**Issue by killo3967** [maforget/ComicRackCE#62]
**Describe the bug**
When updating the program in windows the installation overwrite publisher.zip

Could be posible to not ovewrite the files on this path "C:\Program Files\ComicRack Community Edition\Resources\Icons" if the files are newer?

I added a more complete publisher.zip if you want to update it.

[Publishers.zip](https://github.com/maforget/ComicRackCE/files/14863582/Publishers.zip)

**Comment by maforget** [comment]
There is an option in the installer to not install additional icons and textures. This should do what you want. 

I can't access my computer right now, but I will gladly update the Publisher file the moment I can.

**Comment by maforget** [comment]
Ok regarding the accented characters not appearing in the new version. This is a change from the SharpZipLib in the new versions. The way character encoding as changed from the default. 

Now you can either set character encoding manually when reading:

https://github.com/maforget/ComicRackCE/blob/cb76e9bf6412a10056c64b25fd73fe83063116b7/cYo.Common/Compression/ZipFileFolder.cs#L20-L23

could be changed to:

```c#
public ZipFileFolder(string zipFile)
{
    //437 is the default Zip char Encoding. 850 is the default western windows
    this.zipFile = new ZipFile(zipFile, StringCodec.FromCodePage(437));
}
``` 

Or when creating the Zip file force the UTF-8 flag for the entry and that way it will be detected automatically. This can be achieve (at least with 7-Zip), by either:
1. Set the windows settings to "Beta: Use Unicode UTF-8 for worldwide language support"
2. When creating the zip add `cu` in the Parameters textbox. It will force UTF8 on entries that uses it.

You can confirm the file is correctly created by opening it in 7-Zip and clicking on an entry that had accents and pressing info. Under characteristics you will see _`NTFS : UTF8`_ instead of _`NTFS up`_.

Try this file:  [Publishers.zip](https://github.com/maforget/ComicRackCE/files/14891946/Publishers.zip)
P.S. I've added Glénat to the string, because most of my books are all scrapped under _Glénat_ and not _Éditions Glénat_.

---

Also regarding your initial remark about not overwriting the Publisher file, in addition to the option I mentioned. You can actually have multiples files. The search pattern is _`Publishers*.zip`_, so you could add anything and have `_Publisher_French.zip_` for example. 

You can also add them into your data folder. It supports, alternate configuration (-ac) and portable mode also (-local). Some folder the program will look into:

```
"{InstallDir}\Resources\Icons"
"%ProgramData%\cYo\ComicRack Community Edition\Resources\Icons"
"{DataDir}\Resources\Icons"
```

**Comment by maforget** [comment]
I've updated the Publishers.zip based on yours in fc2947db477d5afde88966710d2bd13e5a70371b, but There are still a lot that might need updating.

I will let this issue open as a place to update the Publishers.zip

**Comment by killo3967** [comment]
I follow your instruction to add "cu" parameter when compressing the folder, but dont work. 
After compress I check that the file "Glenat#Glénat#Glenat Italia#Glénat Spain#Éditions Glénat.png" have the NTFS:UTF8 characteristic and it does. 
After this, I copy the file and rename, the copy, to Éditions Glénat.png and now it works ahd comicRack diskplay the logo.
Could be the "#"?

I don't know why your file works and mine no. 

I checked the headers of both pulishers.zp files and they are complete diferent. What software do you use to compress the folder? I use 7-zip 23.01(x64) in windows 11.

**Comment by maforget** [comment]
I use the same software. 1 thing I've discovered is that you can't rename a file after it is compressed with `cu` it loses UTF8.

And I can confirm it is the **`é`** **`É`**, they are read as **`,`**.

**Comment by killo3967** [comment]
I'm sorry if I didn't make myself clear, but don't rename it afterwards, rename it before recompressing it.
Let's see if this time I explain better. I have taken the file "Glenat#Glénat#Glenat Italia#Glénat Spain#Éditions Glénat.png" and I have duplicated and renamed it to "Éditions Glénat.png", then I have compressed it with the option "cu" and it has shown me the logo in those comics that had the Publisher to "Éditions Glénat", but those that had it for example to "Glenat" or any of its variants has not shown it.

What surprises me is that with the file you sent me it does show the logos in all the variations, that's why I say that either I'm not doing something or the programs or procedures we are using are different.

**Comment by maforget** [comment]
In the file I sent, I've added Glénat to the line. Because you had Glenat (without the é), but not Glénat (with the é), which is what the scrapper returns for most Glénat comics. Maybe that is why mine is working on more comics.

**Comment by maforget** [comment]
Was on mobile, didn't read your message completly. So you have "Glenat#Glénat#Glenat Italia#Glénat Spain#Éditions Glénat.png" along side "Éditions Glénat.png"? It shouldn't change anything, it would take only 1 Éditions Glénat, the first one it encounters.

Here are the settings I've used:
![image](https://github.com/maforget/ComicRackCE/assets/11904426/ed69633c-0f58-4460-ad30-0852406d547d)

A note from what I saw in the 7-zip discussions pages on sourceforge, the setting I mentioned "Beta: Use Unicode UTF-8 for worldwide language support" can change the behavior of 7-zip. Normally it sets it to UTF8 only if it is needed on an entry. Maybe windows code page can affect it also, like western vs eastern.

I could change the program so it uses code page 537, which is the default for zip files and it should be universal for everyone. I just want to test if files with say Japanese Characters would work also with 537.

**Comment by maforget** [comment]
After some test with Japanese characters, i've updated the default Code Page to OEM (437). This should permit wide compatibility with pretty much all Zip software, without any special settings (aka the `cu` parameter).

WIth either 7-zip or WinRAR when using Japanese character it will force it to be UTF8, as there is no choice. Windows will just say that it can't create the zip. Since the Japanese entry is UTF8 it is still read correctly alongside entries like Glénat, which are also read correctly even if they aren't UTF8.

With this change e1a7a1b7e3549436482b54c047f7e2162b39513b all accented character are read correctly regardless of the software used, at least WinRAR, 7-Zip & Windows works correctly.

**Comment by killo3967** [comment]
Tested and now CR could display the logos, anyway i'll use cn option in future.
Thanks very much.

**Comment by maforget** [comment]
I've added some myself. I didn't resize them, because they would loose too much detail and the program doesn't seem to care.

[Publishers.zip](https://github.com/maforget/ComicRackCE/files/14937622/Publishers.zip)

**Comment by ucapato** [comment]
Can use this thread to bring other Publisher Icons to be added in future updates for Publishers.zip?
As I had some self made Publisher Icons to bring in.

I also had some icons for Book Formats as well (e.g: Omnibus, Series instead of the standars Limited Series...)

**Comment by ucapato** [comment]
Hi, 

I created some icons for Format:

Series.png (icon: SER): For Series (So, the original file "Limite Series#Series.png") was renamed to #Limited Series.png")
Omnibus.png (icon: OMN): For Omnibus
Fan Made.png  (icon: FME): For fans that want to make their self art work
Edit.png (icon: ED)

[Formats.zip](https://github.com/maforget/ComicRackCE/files/14971595/Formats.zip)

**Comment by ucapato** [comment]
> I've added some myself. I didn't resize them, because they would lose too much detail and the program doesn't seem to care.
> 
> [Publishers.zip](https://github.com/maforget/ComicRackCE/files/14937622/Publishers.zip)



For this Publishers.zip file, I added a few more from the collection I have:

Globo.png
Marvel Now!.png
Pipoca & Nanquim.png
Rio Gráfica e Editora.png
Webtoon#Line Webtoon.png

[Uploading Publishers.zip…]()

**Comment by maforget** [comment]
> Uploading Publishers.zip…

Your Publishers.zip didn't upload.

**Comment by ucapato** [comment]
> > Uploading Publishers.zip…
> 
> Your Publishers.zip didn't upload.

Sorry, 

Here it goes 

For this Publishers.zip file, I added a few more from the collection I have:

Globo.png
Marvel Now!.png
Pipoca & Nanquim.png
Rio Gráfica e Editora.png
Webtoon#Line Webtoon.png
[Publishers.zip](https://github.com/maforget/ComicRackCE/files/15016888/Publishers.zip)

**Comment by jrft1379** [comment]
Question about multiple publishers: There are plenty of books that have more than one publisher (Amalgam Comics, for example, are from DC and Marvel). How do you get CR to recognize that in the icons? I've tried a couple of methods, but they don't seem to work.

For reference, I have the individual publishers separated by a comma, that way they also show under the individual publisher when I narrow my library down.

I tried to have the icon names match the field name (i.e. DC, Marvel), but I realized that didn't work because CR isn't recognizing them as one entity that gets a special icon.

Also, I've noticed that the "/" character has no replacement value when it comes to the icons. For example, AiT/Planet Lar  won't show up. I've tried substituting the "/" with "-" and just leaving a space instead. Neither worked for displaying the icon.

Any thoughts or ideas for either of these?

**Comment by maforget** [comment]
> Question about multiple publishers: There are plenty of books that have more than one publisher (Amalgam Comics, for example, are from DC and Marvel). How do you get CR to recognize that in the icons? I've tried a couple of methods, but they don't seem to work.

> For reference, I have the individual publishers separated by a comma, that way they also show under the individual publisher when I narrow my library down.

The Publisher field isn't a multi value field like Tags for example. So adding a comma will not give you 2 value just a value that has a comma in it. But when parsing the Publisher filenames it does split it by comma and #. 
https://github.com/maforget/ComicRackCE/blob/f8cf2f46f10d2d781b2030cd9c9a5c365f77574c/ComicRack/Program.cs#L946-L969

The program returns an array of icons when checking them, so technically it might be possible. I am looking at the tags section, it checks for text in the Tags section in the `Special.zip` file.
https://github.com/maforget/ComicRackCE/blob/f8cf2f46f10d2d781b2030cd9c9a5c365f77574c/ComicRack.Engine/ComicBook.cs#L1861-L1943 
https://github.com/maforget/ComicRackCE/blob/f8cf2f46f10d2d781b2030cd9c9a5c365f77574c/ComicRack.Engine/ComicBook.cs#L1841-L1849

Solution may be to have 1 be the Imprint the other the Publisher, have a special icon with the 2 and use the `map.ini` file for special characters. So you could do an icon that has DC & Marvel and call it via the `map.ini` if it has special char so `DC_Comics_Marvel.png = DC Comics, Marvel` as an example. Check the `Formats.zip` for an existent example. Or use the Tags section with a special icon.

I don't know if multiple publisher are very prominent that it would require changing how they work internally, but implementing a way to spit out more than 1 icon, seems to be possible without any UI work.

**Comment by ucapato** [comment]
Hi,

During my Collection Catalog process, I added some more publishers to Publisher.zip. All the previous icons were kept. In addition we have:

Mythos Editora.png
Fox Atomic Comics.png
Graphic India.png
DMG Entertainment#DMG#Valiant Comics#DMG Valiant Entertainment.png
Dom Quixote Publicações.png
Now Comics.png
Avatar Press.png (update) *
Dead Dog.png
Rizzoli Lizard.png
Vecchi editora.png
First Comics#First.png (update)
Ego#Ego Editora.png
Mudnag.png
Super Nova Produções#Super Nova.png
Legendary Comics.png
Farrar, Straus and Giroux#Farrar Straus Giroux#Farrar, Straus, Giroux.png
Delcourt.png (update)
Planeta Cómic.png
Ediouro.png
Simon and Schuster#Simon & Schuster.png
Ten Speed Press.png
Super Genius Comics.png
Skript#Skript Editora.png
Amigo Comics#Amigo.png
Black Mask#Black Mask Studios.png
Great Big Comics.png
Ubisoft#Ubisoft Entertainment.png
Bloch#Bloch Infanto Juvenil.png
Devir Livraria.png
Warren#A Warren Magazine.png
D-Arte#Editora D-Arte.png
Andrews McMeel#Andrews McMeel Publishing.png (update) *
Mythopoeia.png

* Avatar Press and Andrews McMeel already existed in the original Publisher.zip, but they were quite old and with a different color than today's. Maybe I should keep both logs and mark with dates like it is made with DC Comics for example. Need some advise on that.
[Publishers.zip](https://github.com/user-attachments/files/16025355/Publishers.zip)

**Comment by denis-anikin** [comment]
Any reason to update "Zenescope Entertainment"? In the original archive of old ComicRack there was a transparent background, now white

**Comment by denis-anikin** [comment]
Same with "Drawn and Quarterly#Drawn & Quarterly" if I'm not mistaken.
Transparency is gone.

**Comment by ucapato** [comment]
Hi @denis-anikin 

Probably resolution was a bit better at those. But I'll remove the white background and update the zip content.

**Comment by ucapato** [comment]
Here they come,

All the latest additions as previous mentioned, plus the removal background from "Drawn and Quarterly#Drawn & Quarterly" and "Zenescope Entertainment".

[Publishers.zip](https://github.com/user-attachments/files/16030061/Publishers.zip)

**Comment by maforget** [comment]
I've updated the way the build is done in regard to these icons. The icons are not stored as zip files anymore, they are now stored as individual icons instead. So it is way easier to see what as been updated, added or removed. So if you check the commit 0340ab422341a4cc8f1121769d7d4050c63c2d50, you can even see the icons directly on github now. It will make cooperating easier, anyone that wants to contribute can just change the icons in the `Resources\Icons` folder and create a commit. Then you can do a PR, so your contribution will be tracked on github.

The project files will take any folder that is in `Resources\Icons` and add them to the project without any modification. It also support any number of directory that are present in that folder, for future proofing in case we want to add new zips. A [powershell script](https://github.com/maforget/ComicRackCE/blob/master/CompressIconFolders.ps1) will be called during the build and zip them up.

**Comment by ucapato** [comment]
Great @maforget,

Thanks a lot for this way of working to update the Resources\Icon folder.
So, in other words, do we need to use Git knowledge to be able to update in case bringing up a new icon?
I can copy the ComiRackCE Repo to my machine, perform the updates I want, and then add/commit/push to the Main Repository here in Git, isn't that right?
Sorry for the silly question, but I am trying to learn how to use Github productively.

**Comment by ucapato** [comment]
I ended up making a fork of ComicRackCE, so I could add, commit and pushed the proposed Format icons to my own Fork. But I note sure if this is what was exactly proposed here or if there is a better practice.

**Comment by maforget** [comment]
@ucapato Yes that is the proper way to do it. You can use GitHub directly for basic things like uploading new files, renaming. Using a git client for local changes is required for more features. If you don't have or require the whole Visual Studio program just to edit icons, the Github desktop client is pretty good at using the git features with a GUI.

* Clone your fork to a local folder, this is important since if you are downloading the repo only, git won't work.
* It is suggested to keep the master branch clean. If you commit on it directly it will be harder to sync the changes with mine. That can be done via a Fetch on the client or on Github a sync button will appear if there are changes to fetch.
* You should instead create a branch based on the latest master when you want to create a PR. With the client it is easy to switch branches and the files on your computer will change to reflect that.
* With your patch branch checked-out it is only a matter of replacing the files in the folder.
* The client will show you what was modified, deleted, added. You can then create a commit by giving it some description of the change. 
* That branch is now local only, you can do changes or append to that commit while only local. Once you Push it, it will be sent to Github and at that point doing any further changes is not recommended. Instead just push another commit to fix any mistake. 
* Then either on Github or through their client you can create a Pull Request from your patch branch to my master. Any further changes you or I push to your patch branch will show up in the PR.
* I have actions setup so that on PR, once I approve it. A build for that PR will be done and a bot will post a link to it in the PR thread, so it is easier to test that build. On each and every commit you do a the build will be kept updated.
* Once the PR is merged into my master, you can then delete that patch branch if you want. But most just keep it and just do another branch on further changes.

>[!WARNING]
If you do any contribution, I suggest that you make sure the checkbox, keep your email private is enabled in Github settings or your email address will be attached to each and every commit you do. With that setting enabled a github email address proxy will be used instead.

**Comment by ucapato** [comment]
Hi @maforget ,

Appreciate your assistance and patience in explaining how Git and Github work. I am still sometimes lost with the technical words and behavior, but to ride a bike we need to fall sometimes, right? Here we go:

1. I started with the left foot. Had done a fork, clone it into my Client (I am using IntelliJ instead of Visual Studio), and updated the Format folder with my suggested Icons in the Master Branch. Only after that, I have read your advice here.

2. Still, no problem (I guess), I've found some guidelines and I was able to restore my branch by adding the Upstream Remote from your Master, resetting it, and pushing your Master to my Fork again. So now I got my Master's exactly as yours.

3. Now I have created a branch (git checkout -b NewUpdatedIcons in IntelliJ Terminal window) for my Master Branch and desired changes I'll apply to this NewUpdatedIcons branch only.

4. When you mention " In the Github client if you set the fork behavior in the repository settings to contribute to parent, when you create a fork it will be based on my master". How do I achieve that? Or, should I simply click on "Contribute" when adding changes to my NewUpdatedIcons branch like I see in the snapshoot below:

<img width="792" alt="image" src="https://github.com/maforget/ComicRackCE/assets/125318439/853c82be-ec1b-455d-8b8b-ac412980c46d">

**Comment by maforget** [comment]
When I talk about the Github Client I was referring to the Desktop client you can install on your computer. You can do the same thing with your IDE, although it may be more straight forward to use the client. https://github.com/apps/desktop

> In the GitHub client if you set the fork behavior in the repository settings to contribute to parent, when you create a fork it will be based on my master.

The setting I was referring too was this one. 
![Screenshot 2024-07-02 202548](https://github.com/maforget/ComicRackCE/assets/11904426/15f2dcf0-26b3-45ac-b13e-4455227e9cc9)
![Screenshot 2024-07-02 202757](https://github.com/maforget/ComicRackCE/assets/11904426/3771d3b5-fcea-42df-b568-4cbc9c49be89)

It just determines how a new branch will be created either from your master or from mine. If from yours it would be necessary to do what you did in point 2. With the option to contribute, like you can see in the help message on the screenshots, it just creates a branch based on my master. So if I push something new and your master lags behind the new branches will still be based on the newest code. On the website when creating a branch you can choose from with repo/branch to create it. In the Github desktop client a upstream/master is automatically added or your need to do it manually like you did.

Your assumption in point 5 are correct except the contribute option like I talked about earlier. It doesn't do anything until you push something to your repo. At that point it will suggest to merge your NewUpdatedIcons to my master. But you need to push code before that becomes an option. But yes a PR is like asking to add changes to my repo.

Git is very confusing, I am just starting to be somewhat knowledgeable enough to use it effectively. It also doesn't help that all the help all refers to the command line. I am conformable using the command line, but doing something via 2 click in a GUI is easier than having me trying to use a command I will forget anyway. Suggestion I can say is can create as much branches that you want locally and test things out. You can create a backup branch based on another in case you mess up. As long as you don't push code, it stays local.

For example on your error in point 1, where you committed in master. If you didn't push to Github, you could just have done a Reset (mixed keeps your changed file but removes your commit). Stash the changes (keep for later) so that you can change branches and then push those changes to that new branch.

**Comment by ucapato** [comment]
Ok, 

I think I got it now (I hope). I added the new format icons in addition to two new ones (Main Series and Year Zero), set my branch as a contributor to yours, committed, pushed, and did the PR. 

![image](https://github.com/maforget/ComicRackCE/assets/125318439/b6034b65-c1ec-485a-bfc1-baa9234272dc)


Definitely from the Desktop Client, the whole Git process looks much easier, but I'll try my chances (sometimes) in the regular command prompt to see how to do this PR.

Thanks a lot for all the advice and way forward here.

**Comment by chowbok** [comment]
The Andrews-McMeel icon change is a downgrade, IMO. It's an icon -- the text is unnecessary. Also, the transparency has been removed.

**Comment by ucapato** [comment]
> The Andrews-McMeel icon change is a downgrade, IMO. It's an icon -- the text is unnecessary. Also, the transparency has been removed.

Hi @chowbok ,

The latest merge brings the suggested change:

![image](https://github.com/user-attachments/assets/6f2d8bcb-22c7-40dd-9a6f-440a2014e5ab)

**Comment by maforget** [comment]
I've moved the files that where in Resources to `Output/Resources`. So the Publisher directory is now `Output\Resources\Icons\Publishers` for the source. The final directory will still be the same.

The Resources directory now contains the program icons that were previously embedded instead.

**Comment by maforget** [comment]
Just a FYI when adding publisher icons, adding an icon with no date along with those with dates, like in this commit https://github.com/maforget/ComicRackCE/commit/b62cd2c093a8bdf01e90f7ca10716e19e60c7186. The icon without a date should be the latest one. 

The reason is that when adding comics where the date is proposed (pale grey), it will consider the date empty and show the old icon. At least for those that have these dates in proposed value, we should at least show the latest one and not the old one.

**Comment by ucapato** [comment]
> Just a FYI when adding publisher icons, adding an icon with no date along with those with dates, like in this commit [b62cd2c](https://github.com/maforget/ComicRackCE/commit/b62cd2c093a8bdf01e90f7ca10716e19e60c7186). The icon without a date should be the latest one.
> 
> The reason is that when adding comics where the date is proposed (pale grey), it will consider the date empty and show the old icon. At least for those that have these dates in proposed value, we should at least show the latest one and not the old one.

ok,

So I can "invert" the naming convention from the proposed Publisher Icons I have sent, like Dark Horse's Example, today is like:

Dark Horse#Dark Horse Comics.png  ==> Oldest Icon
Dark Horse(1990)#Dark Horse Comics(1990).png
Dark Horse(1991)#Dark Horse Comics(1991).png
Dark Horse(1992-2003)#Dark Horse Comics(1992-2003).png
Dark Horse(2004-2050)#Dark Horse Comics(2004-2050).png ==> Newest Icon

It should be like:
Dark Horse(1986-1988)#Dark Horse Comics(1986-1988).png  ==> Oldest Icon
Dark Horse(1990)#Dark Horse Comics(1990).png
Dark Horse(1991)#Dark Horse Comics(1991).png
Dark Horse(1992-2003)#Dark Horse Comics(1992-2003).png
Dark Horse#Dark Horse Comics.png ==> Newest Icon

I can fix this naming convention for all those Publishers with a date. I'll do that on my next Commit.

**Comment by killo3967** [comment]
Please add Caliber Comics logo when you could.

https://calibercomics.com/

![CaliberLogo](https://github.com/user-attachments/assets/7572a69e-e244-4c80-b80f-d52238397af7)

**Comment by maforget** [comment]
> Please add Caliber Comics logo when you could.

Done

**Comment by killo3967** [comment]
Please add "Best Jackett Press" logo when you could

https://imgur.com/ILJsN7U

![firefox_qbqqGBYSqu](https://github.com/user-attachments/assets/0146638e-e1cf-4980-b101-8d6bc88d62c6)

**Comment by maforget** [comment]
> Please add "Best Jackett Press" logo when you could

done, the winget update should be automated now. Just waiting for approval.

**Comment by killo3967** [comment]
Please add publisher "Standaard Uitgeverij" when you could
![Standaard Uitgeverij](https://github.com/user-attachments/assets/fd5308c3-e01f-445f-9134-1f0879d009e4)

**Comment by maforget** [comment]
> Please add publisher "Standaard Uitgeverij" when you could

Done in b6392f32666f16c53a36f6649c2d6fa0ef25c351

**Comment by killo3967** [comment]
Please add 'Vault' to 'Vault Comics' logo name when you could.

**Comment by chowbok** [comment]
Here are a couple new publisher icons.
 
Educomics.png:
![Image](https://github.com/user-attachments/assets/24099f2d-f1ca-4654-b9d3-60e8fb95c791)

Robin Snyder and Steve Ditko.png:
![Image](https://github.com/user-attachments/assets/3c8be302-da5c-4238-81b5-224cf9c7d193)

Also, could you rename:
- Fawcett Publications.png to Fawcett#Fawcett Publications.png
- Pacific Comics.png to Pacific#Pacific Comics.png
- Vortex Comics.png to Vortex#Vortex Comics.png

Thanks!

**Comment by maforget** [comment]
As discussed here #151, it is now possible to create image packages for any fields. Just name the zip file like the field prefixed with a `_`.

**Comment by chowbok** [comment]
Please rename:

Fox Feature Syndicate.png to Fox Publications#Fox Feature Syndicate.png

Thanks as always!

**Comment by chowbok** [comment]
Another publisher icon.

Fleetway Publications.png:
![Image](https://github.com/user-attachments/assets/eefea4bf-4be5-475e-bfdf-5328e5fc873a)

**Comment by chowbok** [comment]
Mirage Studios.png:
![Image](https://github.com/user-attachments/assets/5d8df17f-9837-4f38-97dc-efb16ee82cfa)

Floating World Comics.png:
![Image](https://github.com/user-attachments/assets/52031c6c-131b-4082-b15e-8abfb663a481)

Yoe Books.png:
![Image](https://github.com/user-attachments/assets/2f42a00e-f358-4611-912f-143eb76f96db)

**Comment by chowbok** [comment]
I'm not great with git, but let me give it a shot.

**Comment by peligwe** [comment]
Action Lab.png 
![Image](https://github.com/user-attachments/assets/c0283d88-4c94-4d4c-adb2-c546f3fa4f5d)

**Comment by peligwe** [comment]
Innovation.png

<img width="120" height="24" alt="Image" src="https://github.com/user-attachments/assets/ca1a2375-031d-4d68-b6cd-5f849666d54c" />

**Comment by maforget** [comment]
Just a note that there is such a thing as too big a size. Even if the program resizes them having a too high resolution image will slow down simply scrolling though while the icon is shown (like in Tiles view). For example the current version of `Soleil Productions.png`.
